dcos_launch/config.py

```diff
@@ -216,7 +225,11 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
         'validator': validate_url},
     'template_parameters': {
         'type': 'dict',
-        'required': True}}
+        'required': True},
+    'zen_helper': {
```

Comment:
zen_helper is not a common template deployment feature and should not be inherited

Reply:
@mellenburg It used to be in the COMMON_SCHEMA, is that better?

Reply:
Done